src/plugins/github/githubGitGateway.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import {type RepoId, repoIdToString} from "../../core/repoId";
+import type {Hash} from "../git/types";
+import type {GitGateway, URL} from "../git/gitGateway";
+
+export class GithubGitGateway implements GitGateway {
+  commitUrl(repoId: RepoId, hash: Hash): URL {
+    const repoString = repoIdToString(repoId);
+    return `https://github.com/${repoString}/commit/${hash}`;
```

Comment:
Hmm… `repoIdToString` doesn’t guarantee any particular representation. If I were changing that module, I would feel free to change its representation to `(rid) => stringify(rid)`.

What do you think about using `` `https://github.com/${repoId.owner}/${repoId.name}/commit/${hash}` `` here instead?